verticapy/tests/utilities/test_utilities.py

```diff
@@ -359,6 +359,10 @@ def test_vdf_from_relation(self, base):
         )
         assert result["verticapy test *+"].avg() == 1.0
 
+    # TODO
```

Comment:
Instead of having a fake passing test, it would be better to mark it with `skip` decorator. Then, it would be quite easy to monitor which tests are skipped and why.
You may replace `# TODO` with `@pytest.mark.skip(reason="this test will be implemented later")`